repository-hbase/src/main/java/hbase/load/BulkLoader.java

```diff
@@ -97,11 +107,40 @@ public int run(String[] strings) throws Exception {
         }
     }
 
-    private int load(String tableNameStr, String referencePeriod, String inputFile) {
+    private int load(String tableNameStr, String referencePeriod, String inputFile) throws Exception {
         return load(tableNameStr, referencePeriod, inputFile, "");
     }
 
-    private int load(String tableNameStr, String referencePeriod, String inputFile, String outputFilePath) {
+    private void getHeader(String inputFile, Configuration conf) throws Exception {
+        Path path = new Path(inputFile);
+        FileSystem fs = FileSystem.get(path.toUri(), conf);
+
+        try( BufferedReader readFile = new BufferedReader(new InputStreamReader(fs.open(path))) ) {
```

Comment:
InputStreamReader not being closed?

Change to...

try( 
         InputStreamReader  reader = new InputStreamReader(fs.open(path));
 BufferedReader readFile = new BufferedReader(reader)
    )